packages/ui/src/utils/UiHelpers.ts

```diff
@@ -63,3 +63,27 @@ export async function preloadImage(src: string) {
     image.src = src
   })
 }
+
+export function useScript(url: string) {
+  const script = document.createElement('script')
+
+  script.src = url
+  script.async = true
+
+  document.body.appendChild(script)
+
+  return () => {
+    document.body.removeChild(script)
+  }
+}
+
+export const ZorbPackageScript =
```

Comment:
Any 3rd party lib we use, should come from npm. If it is not available there, you can copy source to utils i.e `Zorb.ts` (see how this was done for Whatmesh). And then import it where it is required (see if you can strip it down as well for unused code). This means that `useScript` util will also not be required.

Reply:
Removed + rebuilt with one small `tinycolors` library !